diff --git a/node_modules/ical-generator/index.d.ts b/node_modules/ical-generator/index.d.ts
index 08210ea..0186893 100644
--- a/node_modules/ical-generator/index.d.ts
+++ b/node_modules/ical-generator/index.d.ts
@@ -81,7 +81,7 @@ declare module 'ical-generator' {
       timezone?: string;
       recurrenceId?: moment.Moment | Date | string;
       transparency?: string;
-      x: { key:string, value:string }[];
+      x?: { key:string, value:string }[];
     }
 
     interface RepeatingData {
