diff --git a/node_modules/rrule/dist/esm/src/rrule.d.ts b/node_modules/rrule/dist/esm/src/rrule.d.ts
index c9da82f..20e27e5 100644
--- a/node_modules/rrule/dist/esm/src/rrule.d.ts
+++ b/node_modules/rrule/dist/esm/src/rrule.d.ts
@@ -23,7 +23,7 @@ export declare const defaultKeys: ("freq" | "dtstart" | "interval" | "wkst" | "c
  *        The only required option is `freq`, one of RRule.YEARLY, RRule.MONTHLY, ...
  * @constructor
  */
-export default class RRule implements QueryMethods {
+export class RRule implements QueryMethods {
     _cache: Cache | null;
     origOptions: Partial<Options>;
     options: ParsedOptions;
